shoop/xtheme/resources.py

```diff
@@ -132,7 +132,7 @@ def _render_resource(self, resource):
 
         # TODO: should this be extensible?
 
-        if resource.endswith(".js"):
+        if resource.endswith(".js") or resource.endswith("/jsi18n/"):
```

Comment:
This doesn't look like a good idea. Why not make the url something like `/i18n.js`?